S3TransferUtility-Sample/Swift/S3TransferUtilitySampleSwiftUITests/S3TransferUtilitySampleSwiftUITests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  S3TransferUtilitySampleSwiftUITests.swift
+//  S3TransferUtilitySampleSwiftUITests
+//
+//  Created by Kumar, Roshan on 12/7/18.
+//  Copyright © 2018 Amazon. All rights reserved.
+//
+
+import XCTest
+
+class S3TransferUtilitySampleSwiftUITests: XCTestCase {
+        
+    override func setUp() {
+        super.setUp()
+        
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIApplication().launch()
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testExample() {
+        // Use recording to get started writing UI tests.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        XCUIApplication().buttons["Select Image and Upload"].tap()
```

Comment:
What features is this testing? We should test for a successful upload / download and verify if the image was uploaded/ downloaded correctly. Tapping the button will not indicate a failure/ success of the operation.